presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestHiveWindowQueries.java

```diff
@@ -23,9 +23,9 @@ public TestHiveWindowQueries()
         super(true);
     }
 
-    @Test (enabled = false)
+    @Test
     public void testWindow()
     {
-        assertQueryFails("SELECT clerk, orderdate, orderkey, totalprice, rank() OVER (PARTITION BY clerk ORDER BY totalprice) AS rnk FROM orders ORDER BY clerk, rnk", ".*Unknown plan node type com.facebook.presto.sql.planner.plan.WindowNode.*");
+        assertQuery("SELECT clerk, orderdate, orderkey, totalprice, rank() OVER (PARTITION BY clerk ORDER BY totalprice) AS rnk FROM orders ORDER BY clerk, rnk");
```

Comment:
Would you add queries for all currently supported functions?

Reply:
I have added one query per window function supported now. Though I have plans to add more exhaustive tests in the future.